playground/index.html

```diff
@@ -104,7 +103,7 @@
           <option value="javascript">js</option>
           <option value="ocaml">ocaml</option>
           <option value="python">python</option>
-          <option value="v">v</option>
+          <option value="text">v</option> <!-- There is no syntax highlighting for V lang. Therefore, we use "text" -->
```

Comment:
Should we use Go for V? The languages are very similar.

Reply:
What is "V lang" anyway? :)
It seems to me that Go highlighting doesn't work well for this strange unicorn of a language.
I would leave it like that. But we can change if you want.

Reply:
https://vlang.io 

Ok, let's leave it as is.